db/seed.sql

```diff
@@ -108,7 +108,7 @@ CREATE TABLE `admin_user` (
_express_checkout_settings_payflow_link_express_checkout_advanced\";s:1:\"0\";s:45:\"payment_us_paypal_alternative_payment_methods\";s:1:\"1\";s:65:\"payment_us_paypal_alternative_payment_methods_express_checkout_us\";s:1:\"0\";s:91:\"payment_us_paypal_alternative_payment_methods_express_checkout_us_express_checkout_required\";s:1:\"1\";s:134:\"payment_us_paypal_alternative_payment_methods_express_checkout_us_express_checkout_required_express_checkout_required_express_checkout\";s:1:\"1\";s:105:\"payment_us_paypal_alternative_payment_methods_express_checkout_us_express_checkout_required_advertise_bml\";s:1:\"0\";s:127:\"payment_us_paypal_alternative_payment_methods_express_checkout_us_express_checkout_required_advertise_bml_settings_bml_homepage\";s:1:\"0\";s:131:\"payment_us_paypal_alternative_payment_methods_express_checkout_us_express_checkout_required_advertise_bml_settings_bml_categorypage\";s:1:\"0\";s:130:\"payment_us_paypal_alternative_payment_methods_express_checkout_us_express_checkout_required_advertise_bml_settings_bml_productpage\";s:1:\"0\";s:127:\"payment_us_paypal_alternative_payment_methods_express_checkout_us_express_checkout_required_advertise_bml_settings_bml_checkout\";s:1:\"0\";s:77:\"payment_us_paypal_alternative_payment_methods_express_checkout_us_settings_ec\";s:1:\"1\";s:98:\"payment_us_paypal_alternative_payment_methods_express_checkout_us_settings_ec_settings_ec_advanced\";s:1:\"0\";s:133:\"payment_us_paypal_alternative_payment_methods_express_checkout_us_settings_ec_settings_ec_advanced_express_checkout_billing_agreement\";s:1:\"0\";s:133:\"payment_us_paypal_alternative_payment_methods_express_checkout_us_settings_ec_settings_ec_advanced_express_checkout_settlement_report\";s:1:\"0\";s:124:\"payment_us_paypal_alternative_payment_methods_express_checkout_us_settings_ec_settings_ec_advanced_express_checkout_frontend\";s:1:\"0\";s:24:\"payment_us_purchaseorder\";s:1:\"0\";s:34:\"payment_us_authorizenet_directpost\";s:1:\"0\";s:13:\"oauth_cleanup\";s:1:\"1\";s:14:\"oauth_consumer\";s:1:\"1\";s:12:\"admin_emails\";s:1:\"1\";s:13:\"admin_startup\";s:1:\"1\";s:9:\"admin_url\";s:1:\"1\";s:14:\"admin_security\";s:1:\"1\";s:15:\"admin_dashboard\";s:1:\"1\";s:13:\"admin_captcha\";s:1:\"1\";s:11:\"system_cron\";s:1:\"1\";s:11:\"system_smtp\";s:1:\"1\";s:15:\"system_currency\";s:1:\"1\";s:24:\"system_adminnotification\";s:1:\"1\";s:13:\"system_backup\";s:1:\"1\";s:22:\"system_full_page_cache\";s:1:\"1\";s:34:\"system_media_storage_configuration\";s:1:\"1\";s:17:\"system_cron_index\";s:1:\"0\";s:19:\"system_cron_default\";s:1:\"0\";s:30:\"system_full_page_cache_varnish\";s:1:\"0\";s:34:\"dev_front_end_development_workflow\";s:1:\"1\";s:12:\"dev_restrict\";s:1:\"1\";s:9:\"dev_debug\";s:1:\"1\";s:12:\"dev_template\";s:1:\"1\";s:20:\"dev_translate_inline\";s:1:\"1\";s:6:\"dev_js\";s:1:\"1\";s:7:\"dev_css\";s:1:\"1\";s:9:\"dev_image\";s:1:\"1\";s:10:\"dev_static\";s:1:\"1\";s:8:\"dev_grid\";s:1:\"1\";}}',NULL,NULL,'en_US',0,NULL,NULL);
```

Comment:
Only the 2nd, 3rd and 4th fields have changed (first name, last name, email address).